src/Android/Xamarin.Android/Samples/Data/GeodatabaseTransactions/GeodatabaseTransactions.cs

```diff
@@ -339,12 +339,16 @@ private void GdbTransactionStatusChanged(object sender, TransactionStatusChanged
         private void BeginTransaction(object sender, EventArgs e)
         {
             // See if there is a transaction active for the geodatabase
-            if (!_localGeodatabase.IsInTransaction)
+            if (_localGeodatabase!= null && !_localGeodatabase.IsInTransaction)
             {
                 // If not, begin a transaction
                 _localGeodatabase.BeginTransaction();
                 _messageTextBlock.Text = "Transaction started";
             }
+            else
+            {
+                _messageTextBlock.Text = "Not yet ready for new transaction.";
+            }
```

Comment:
The sample seems a little flakey about initializing / getting data. It's good to have this check.